lib/models/get-file-content.ts

```diff
@@ -6,7 +6,7 @@ export type GetFileContentOptionsType = { encoding: string | null } | null;
 /**
  * Return string if encoding is string and Buffer otherwise.
  */
-export type GetFileContentType = (filePath: string, options?: GetFileContentOptionsType) => Promise<any>;
+export type GetFileContentType = (filePath: string, options?: GetFileContentOptionsType) => Promise<string | Buffer>;
```

Comment:
As `fs.readFile` is used both types can be returned. The main problem is that the code now assumes it is a string in some places and a Buffer in others. Not sure if this is the way to go, but as this is a 'public API' I don't know if we can change it.

Reply:
This looks like the correct change to me. This method is used to fetch both binary files and text based files from the component set, IMO not something we want to change at this point. Downside is indeed that you need to cast when using it.